docs/getting-started/creating-plugins.md

```diff
@@ -35,6 +41,53 @@ The `json` file contains the following structure:
 }
 ```
 
+#### Definition structure
+
+The `Definition` property has the following structure:
+
+```json
+{
+  "Id": "Id shall contain a stable identifier for the rule",
+  "Name": "Name may contain an identifier that is understandable to an end user",
+  "HelpUri": "HelUri may contain the absolute URI of the primary documentation for the reporting item",
+  "Message": "Message may contain the default message that will be used",
+  "Kind": "ResultKind of the issue",
```

Comment:
>ResultKind [](http://example.com/codeflow?start=11&length=10)

Whenever you reference a term like 'ResultKind' or 'FailureLevel', you should define it and also consider describing candidate values. Otherwise you really haven't communicated any additional information.

'The severity, or SARIF 'FailureLevel' that is associated with this rule (e.g., 'Error', Warning' or 'Note').

I leave the string for 'ResultKind' as an exercise for you. :)